x-pack/plugins/infra/public/components/source_configuration/fields_configuration_panel.tsx

```diff
@@ -57,7 +59,12 @@ export const FieldsConfigurationPanel = ({
         />
       }
     >
-      <EuiFieldText fullWidth disabled={isLoading} isLoading={isLoading} {...timestampFieldProps} />
+      <EuiFieldText
+        fullWidth
+        disabled={isLoading || disabled}
```

Comment:
I wonder if using the `readOnly` prop instead of disabling the field would be appropriate :thinking: is there precedent in other parts of Kibana for `disabled` over `readOnly` if the permission is not sufficient?

Reply:
I was unaware of the option to use `readOnly`, but that makes much more sense here, I will make this change. Thanks Felix!